### ISSUE TITLE
After `Pkg.update()`, TikzPictures won't render in IJulia notebooks

### ISSUE THREAD

**Issue by tawheeler** [sisl/TikzPictures.jl#11]
@rtompa2 originally noticed the problem.
I get the following errror:

```
Error saving as SVG
SystemError: opening file tikzpicture.log: No such file or directory

 in open at ./iostream.jl:90
 in open at ./iostream.jl:112
 in readall at io.jl:266
 in save at /home/tim/.julia/v0.4/TikzPictures/src/TikzPictures.jl:160
 in save at /home/tim/.julia/v0.4/TikzPictures/src/TikzPictures.jl:210
 in writemime at /home/tim/.julia/v0.4/TikzPictures/src/TikzPictures.jl:242
 in writemime at /home/tim/.julia/v0.4/PGFPlots/src/PGFPlots.jl:571
 in sprint at iostream.jl:206
 in display_dict at /home/tim/.julia/v0.4/IJulia/src/execute_request.jl:29
```

Could this be a problem with IJulia or Interact updating?

**Comment by mykelk** [comment]
It seems to work for me. Can you post your `Pkg.status()` along with the commands that led to this error?

**Comment by rtompa2** [comment]
So I tried to run the following: 
yyy = [4,5,6]
xxx = [1,2,3]
plot(xxx,yyy)

I got the following message:
Error saving as SVG
SystemError: opening file tikzpicture.log: No such file or directory

My Pkg.status() is as follows:
17 required packages:
 - Colors                        0.6.2
 - Distances                     0.3.0
 - Gadfly                        0.4.1
 - Geodesy                       0.0.1
 - IJulia                        1.1.8
 - ImageMagick                   0.1.2
 - Interact                      0.3.0
 - JLD                           0.5.7+             master
 - NetCDF                        0.3.0
 - ODE                           0.2.1
 - PGFPlots                      1.3.0+             master
 - ProfileView                   0.1.1
 - ProgressMeter                 0.2.3
 - PyPlot                        2.1.1
 - StatsBase                     0.7.4
 - Winston                       0.11.13
 - ZMQ                           0.3.1
60 additional packages:
 - ArrayViews                    0.6.4
 - BinDeps                       0.3.20
 - Blosc                         0.1.4
 - Cairo                         0.2.31
 - Calculus                      0.1.14
 - Codecs                        0.1.5
 - ColorTypes                    0.2.0
 - ColorVectorSpace              0.1.1
 - Compat                        0.7.8
 - Compose                       0.4.1
 - Conda                         0.1.8
 - Contour                       0.0.8
 - DataArrays                    0.2.20
 - DataFrames                    0.6.10
 - DataStructures                0.4.2
 - Dates                         0.4.4
 - DiscreteMDPs                  0.0.0-             master (unregistered)
 - Distributions                 0.8.9
 - Docile                        0.5.23
 - DualNumbers                   0.2.1
 - FileIO                        0.0.3
 - FixedPointNumbers             0.1.1
 - Formatting                    0.1.5
 - GZip                          0.2.18
 - Graphics                      0.1.3
 - Grid                          0.4.0
 - GridInterpolations            0.0.2              master
 - Gtk                           0.9.3
 - GtkUtilities                  0.0.7
 - HDF5                          0.5.7
 - Hexagons                      0.0.4
 - Homebrew                      0.2.0
 - Images                        0.5.2
 - ImmutableArrays               0.0.11
 - IniFile                       0.2.5
 - Iterators                     0.1.9
 - JSON                          0.5.0
 - KernelDensity                 0.1.2
 - LaTeXStrings                  0.1.6
 - Loess                         0.0.6
 - Measures                      0.0.2
 - NaNMath                       0.1.1
 - Nettle                        0.2.1
 - Optim                         0.4.4
 - PDMats                        0.3.6
 - Polynomials                   0.0.5
 - PyCall                        1.2.0
 - Reactive                      0.3.0
 - Reexport                      0.0.3
 - SHA                           0.1.2
 - SIUnits                       0.0.6
 - Showoff                       0.0.6
 - SortingAlgorithms             0.0.6
 - StatsFuns                     0.2.0
 - TexExtensions                 0.0.3
 - TikzPictures                  0.3.0+             master
 - Tk                            0.3.7
 - URIParser                     0.1.2
 - WoodburyMatrices              0.1.4
 - Zlib                          0.1.12

**Comment by tawheeler** [comment]
Also outside of IJulia:

```
julia> using PGFPlots

julia> plot([1,2,3], [4,5,6])
TikzPictures.TikzPicture("\\begin{axis}[view = {0}{90}]\\addplot+ coordinates {\n(1, 4)\n(2, 5)\n(3, 6)\n};\n\\end{axis}\n","","\\usepackage{pgfplots}\n\\pgfplotsset{compat=newest}\n\\pgfplotsset{every axis legend/.append style={%\ncells={anchor=west}}\n}\n\\usepgfplotslibrary{polar}\n\\usetikzlibrary{arrows}\n\\tikzset{>=stealth'}\n",true,true)

julia> save("hi.tex", plot([1,2,3], [4,5,6]))

julia> save("hi.pdf", plot([1,2,3], [4,5,6]))
ERROR: SystemError: opening file hi.log: No such file or directory
 in open at ./iostream.jl:90
 in open at ./iostream.jl:112
 in readall at io.jl:266
 in save at /home/tim/.julia/v0.4/TikzPictures/src/TikzPictures.jl:160
 in save at /home/tim/.julia/v0.4/PGFPlots/src/PGFPlots.jl:580

julia> save("hi.svg", plot([1,2,3], [4,5,6]))
Error saving as SVG
ERROR: SystemError: opening file hi.log: No such file or directory
 in open at ./iostream.jl:90
 in open at ./iostream.jl:112
 in readall at io.jl:266
 in save at /home/tim/.julia/v0.4/TikzPictures/src/TikzPictures.jl:160
 in save at /home/tim/.julia/v0.4/TikzPictures/src/TikzPictures.jl:210
 in save at /home/tim/.julia/v0.4/PGFPlots/src/PGFPlots.jl:582
```

**Comment by mykelk** [comment]
Fixed with 3d7c404810b9f015f50331ae0dc61eefbf9bc31f.
